app/reducers/lnd.js

```diff
@@ -87,11 +88,18 @@ export const lndSyncStatus = (event, status) => async (dispatch, getState) => {
   }
 }
 
+export const startLightningGrpc = () => dispatch => dispatch(setLoading(true))
+export const startLightningGrpcFinished = () => dispatch => dispatch(setLoading(false))
+
+export const startWalletUnlockerGrpc = () => dispatch => dispatch(setLoading(true))
```

Comment:
Triggering the loading screen on and off like this causes a bit of a flicker effect.

For example:

On the home screen when clicking the "launch: button to launch a local wallet, the loading screen flickers on and off in the split second between when you click the button, and when the password field shows, whilst the wallet unlocker service starts up.

I don't think we really want to trigger the loading screen to show here. Better that the loading button just goes to the processing state until we are ready to show the password entry form.